### ISSUE TITLE
Wrong links in docs

### ISSUE THREAD

**Issue by eugenesvk** [mdevils/rust-osakit#1]
link to value on this page https://docs.rs/osakit/latest/osakit/type.Value.html
opens an invalid 
https://docs.rs/serde_json/1.0.133/x86_64-apple-darwin/serde_json/value/enum.Value.html

which redirects to an invalid search query

which should instead be 
https://docs.rs/serde_json/1.0.133/serde_json/enum.Value.html

since serde_json doesn't seem to have platform-dependend values?

**Comment by mdevils** [comment]
Hello @eugenesvk. I believe these docs are generated by `docs.rs` itself. Do you have any idea why it fails in this case?

**Comment by eugenesvk** [comment]
I think it fails because there is no `x86_64-apple-darwin` platform in the serde crate while your crate has it.

Though no idea whether it's possible to configure docs to link to non-platform-specific docs in a dependency

**Comment by mdevils** [comment]
This is how these links defined in `osakit`:

```rust
/// [`serde_json::Value`] from [`serde_json`].
pub type Value = serde_json::Value;
/// [`serde_json::Number`] from [`serde_json`].
pub type Number = serde_json::Number;
/// [`serde_json::Map`] from [`serde_json`].
pub type Map<K, V> = serde_json::Map<K, V>;
/// [`serde_json::from_value`] from [`serde_json`].
pub use serde_json::from_value;
/// [`serde_json::to_value`] from [`serde_json`].
pub use serde_json::to_value;
```

When generating docs locally, everything works fine.

Looks like this is a bug of https://docs.rs.

**Comment by eugenesvk** [comment]
yeah, locally as far as I understand, it doesn't generate any platforms except the one you compile it on, you don't have the platform picker, so it's not an issue
